resources/leiningen/new/trapperkeeper/src/web_service.clj

```diff
@@ -14,8 +15,10 @@
     (let [url-prefix (get-route this)]
       (add-ring-handler
         this
-        (compojure/context url-prefix []
-          (core/app (tk-services/get-service this :HelloService))))
+        (->> (core/app (tk-services/get-service this :HelloService))
+             (comidi/context url-prefix)
+             (comidi/routes->handler)
+             (wrap-params)))
```

Comment:
`wrap-params` was not in the previous template. It is also unnecessary as we have no parameters in the example services. Should we lose this?